convert2rhel/actions/post_conversion/modified_rpm_files_diff.py

```diff
@@ -0,0 +1,82 @@
+# Copyright(C) 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import difflib
+import logging
+import os
+
+from convert2rhel import actions, utils
+from convert2rhel.logger import LOG_DIR
+from convert2rhel.systeminfo import system_info
+from convert2rhel.toolopts import POST_RPM_VA_LOG_FILENAME, PRE_RPM_VA_LOG_FILENAME
+
+
+logger = logging.getLogger(__name__)
+
+
+class ModifiedRPMFilesDiff(actions.Action):
+    id = "MODIFIED_RPM_FILES_DIFF"
+
+    def run(self):
+        """
+        Get a list of modified rpm files after the conversion and
+        compare it to the one from before the conversion.
+        """
+        super(ModifiedRPMFilesDiff, self).run()
+
+        logger.task("Final: Show RPM files modified by the conversion")
+
+        system_info.generate_rpm_va(log_filename=POST_RPM_VA_LOG_FILENAME)
+
+        pre_rpm_va_log_path = os.path.join(LOG_DIR, PRE_RPM_VA_LOG_FILENAME)
+        if not os.path.exists(pre_rpm_va_log_path):
+            logger.info("Skipping comparison of the 'rpm -Va' output from before and after the conversion.")
+            self.add_message(
+                level="INFO",
+                id="SKIPPED_MODIFIED_RPM_FILES_DIFF",
+                title="Skipped comparison of 'rpm -Va' output from before and after the conversion.",
+                description="Comparison of 'rpm -Va' output was skipped due missing output "
+                "of the 'rpm -Va' run before the conversion.",
+                diagnosis="This is caused mainly by using '--no-rpm-va' argument for convert2rhel.",
+            )
+            return
```

Comment:
Just to avoid having different messages in the waring and the message.

```suggestion
            title = "Skipping comparison of the 'rpm -Va' output from before and after the conversion."
            logger.info(title)
            self.add_message(
                level="INFO",
                id="SKIPPED_MODIFIED_RPM_FILES_DIFF",
                title=title,
                description="Comparison of 'rpm -Va' output was skipped due missing output "
                "of the 'rpm -Va' run before the conversion.",
                diagnosis="This is caused mainly by using '--no-rpm-va' argument for convert2rhel.",
            )
            return
```